js/interface.js

```diff
@@ -2536,6 +2551,14 @@ $('[data-toggle="tooltip"]').tooltip({
   delay: { "show": 500, "hide": 300 }
 });
 
+$('[data-template="fl-store-releaseNotes"]').on('click', function() {
```

Comment:
The iframe should not be navigating to `#`.

```suggestion
$('[data-template="fl-store-releaseNotes"]').on('click', function(e) {
  e.preventDefault();
```

Reply:
@tonytlwu, fixed